lib/ops/ops.go

```diff
@@ -759,35 +762,49 @@ func (r RotateSecretsRequest) SiteKey() SiteKey {
 	}
 }
 
-// RotatePlanetConfigRequest is a request to rotate server's planet configuration package
-type RotatePlanetConfigRequest struct {
-	// AccountID is the account id of the local cluster
-	AccountID string `json:"account_id"`
-	// ClusterName is the local cluster name
-	ClusterName string `json:"cluster_name"`
-	// OperationID is the id of the operation
-	OperationID string `json:"operation_id"`
+// Check validates this request
+func (r RotateTeleportConfigRequest) Check() error {
+	if err := r.Key.Check(); err != nil {
+		return trace.Wrap(err)
+	}
+	return nil
+}
+
+// RotateTeleportConfigRequest is a request to rotate teleport server's configuration package
+type RotateTeleportConfigRequest struct {
+	// Key identifies the cluster operation
+	Key SiteOperationKey `json:"key"`
 	// Server is the server to rotate configuration for
 	Server storage.Server `json:"server"`
 	// Servers is all cluster servers
-	Servers []storage.Server `json:"servers"`
+	Servers storage.Servers `json:"servers"`
+	// Node specifies the configuration package for teleport nodes
+	Node *loc.Locator `json:"node,omitempty"`
```

Comment:
I left out the changes for the master teleport configuration since it's out of scope for offline generation.